src/app/themes/volumio/components/volume-manager/volumio-volume-manager.html

```diff
@@ -33,7 +33,7 @@
 
 <div ng-if="volumeManager.matchmediaService.isPhone" class="volumeManager">
   <div class="volumeSliderWrapper" ng-show="volumeManager.showVerticalSlider">
-    <div class="bigText">{{volumeManager.playerService.volume}}</div>
```

Comment:
keep .volume

Reply:
The text is meant to be an indicator of the slider's value. Isn't it more logically correct if the text mirrors the direct value of the slider, which in term mirrors the volume of the state, rather then them both mirroring the state?

From a functional perspective, the problem is that the text would only update with the slider's value after the BE has reacted to the change, which means lagg.